ftp-ballerina/tests/resources/public.key

```diff
@@ -0,0 +1 @@
+-----BEGIN PUBLIC KEY-----
MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAsmSZBX2+7/8TAMQyarFO
YNDWjS7aIg48vr4EduNmapoYBU+SPt77k5T1A2AIZ+A7Z1DEZ+JZHpZeO4j1hu5h
pvRxl4XZtvaplX5JmVTWoc2Msb9i0y3NLh1A37LDsxrlLuQGxPWQaP+BUGhBKQ3s
75wqu4U2MVSXbz5d1THBZY8d1Zt/CpfAGoutfNtoL1eI4L1HLqwn0G1BlN5oc1de
ZFAzIqTk5YhhGVSabi5dnQTSkb4370ombDNg5FojEJx7TNTuVQYI482hTC34nVoQ
XI3YFpJitwMbLKKIRUzDHk2RbnVfh7LBNmvedcAZR8DotIrcW4MC/hKzPxMp3u7p
jXSRuhVIW3FupGD1jW0EXFqXnNKgSgIXkysZjSrh3RO6Ae6uFIlZL1wb6TfqpXzJ
RyV4Ut+Nx3t00LWdpleM4Q8B7luJ1cLcYEAZ8a/Nkdi3IBGGKt+hLWYbIpF/4FQb
4YFeBVcwP1SyU8TrYoTRxPVtzo6e/FjKy6YXnnNtrQjIUOkvYY6dh/rmwJpU3BCS
zMGhgY7pfwdc0rq2GS5iOuokkdqpE3I+p8gcqXQa0O0VSKtIjKECmFe/X4pifPO1
BFbmef5ObBrBpashUd1Sc7nRYpNZGHvJnNdkDKX9hSt6yO3vttjspBKIDGQGqD9g
Cs2oLFLOHHbaYwYJtUlMd/sCAwEAAQ==
-----END PUBLIC KEY-----
```

Comment:
Shall we convert this key into multi-line format? Also a new line at the EOF.

Reply:
Added EOF. This is already in the multi-line format. You can see it if you click on `View file` in the `Files changed` view.